scripts/ci/test_init.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from util import SRC_PATH
+import logging
+import os
+import sys
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+logger.addHandler(ch)
+
+
+check_path = 'vendored_sdks'
+
+
+def check_init_files():
+    """ Check if the vendored_sdks directory contains __init__.py in all extensions """
+    ref = []
+    # SRC_PATH: azure-cli-extensions\src
+    for src_d in os.listdir(SRC_PATH):
```

Comment:
Is it better to only check the extension that the PR is working on?

Reply:
`os.walk` is very fast, it takes less than six seconds to check all the files, and the incremental check takes more time and effort.